Dimensions.java

```diff
@@ -0,0 +1,50 @@
+package howdryiam;
+
+public class Dimensions {
+  private double width = 0;
+  private double height = 0;
+  private double depth = 0;
+  private String units = "in";
+
+  /**
+   * Constructs a dimension of a 3d box
+   * @param width
+   * @param height
+   * @param depth
+   * @param units 
+   */
+  public Dimensions(double width, double height, double depth, String units) {
+	this.depth = depth;
+	this.height = height;
+	this.units = units;
+	this.width = width;
+  }
```

Comment:
I like this second constructor where you can change the "units" if you wanted. Did you test it to see if it worked?